valhalla/baldr/time_info.h

```diff
@@ -140,18 +140,18 @@ struct TimeInfo {
       return {false, 0, 0, kInvalidSecondsOfWeek, 0, false, nullptr};
     }
     const auto then_date = date::make_zoned(tz, parsed_date, date::choose::latest);
-    uint64_t local_time = date::to_utc_time(then_date.get_sys_time()).time_since_epoch().count();
+    uint64_t sys_epoch = then_date.get_sys_time().time_since_epoch().count();
```

Comment:
this is the only real change in this PR. by converting with `utc_clock` here instead of `system_clock` we added leap seconds. this TimeInfo member is later used in various places, e.g. when checking conditional access restrictions where we convert it again to `system_clock` (keeping the leap seconds), then using that sys time in `make_zoned` which again adds leap seconds. so it's better to just keep it as sys time in the first place, not UTC.